README.md

```diff
@@ -605,9 +609,11 @@ automatically box (encrypt) the message `content` if the appropriate
 - break (because it should know how to handle `recps`, but can't), and so throw an `Error`
 
 ## db.addUnboxer: sync
+
 ```js
 db.addUnboxer({ key: unboxKey, value: unboxValue, init: initBoxer })
 ```
+
```

Comment:
this should probably change to `initUnboxer` for clarity (now that we have another `initBoxer`